man/man5/zfs-module-parameters.5

```diff
@@ -854,8 +854,9 @@ Default value: \fB10\fR%.
 \fBzfs_arc_max\fR (ulong)
 .ad
 .RS 12n
-Max arc size of ARC in bytes. If set to 0 then it will consume 1/2 of system
-RAM. This value must be at least 67108864 (64 megabytes).
+Max size of ARC in bytes. If set to 0 then it will consume at most 1/2 of system
+RAM on Linux or the larger of 5/8 of system RAM and all system RAM - 1GB on
+FreeBSD. This value must be at least 67108864 (64 megabytes).
```

Comment:
How about.

"Max size of ARC in bytes. If set to 0 then it will consume at most 1/2 of system
RAM on Linux, for FreeBSD the larger of 5/8 or all of system RAM - 1GB will be
used.  This value must be at least 67108864 (64 megabytes)."

Reply:
It still seems cumbersome. What about this:
```
Max size of ARC in bytes.  If set to 0 then the max size of ARC is determined
by the amount of system memory installed.  For Linux, 1/2 of system memory will
be used as the limit.  For FreeBSD, the larger of all system memory - 1GB or
5/8 of system memory will be used as the limit.  This value must be at least
67108864 (64 megabytes).
```

Reply:
Even better.  Let's go with yours.